test/acceptance/graphql_resolvers/metrics_stability_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	graphqlhelper "github.com/weaviate/weaviate/test/helper/graphql"
 )
 
+const metricClassPrefix = "metricsclassprefix"
```

Comment:
Weaviate will automatically uppercase the first letter of the class (requirement for class names to start uppercased). Tests are failing during the query phase right now because GraphQL doesn't recognize the lowercased class. If you turn this into

```go
const metricClassPrefix = "Metricsclassprefix"
```

tests should be green.